index.js

```diff
@@ -0,0 +1,37 @@
+window.addEventListener('DOMContentLoaded', () => {
+
+    const popupImages = document.querySelectorAll('.featured-img__style');
+    const popup = document.querySelector('.popup');
+
+    popupImages.forEach((img, index) => {
+        img.addEventListener('click', (img) => {

```

Comment:
What have you did here?

Reply:
Thanks for question!
added a "click" event for each image